src/Nethermind/Nethermind.Logging.NLog/NLogLogger.cs

```diff
@@ -45,27 +41,32 @@ public NLogLogger(string loggerName = null)
 
         public void Info(string text)
         {
-            _logger.Info(text);
+            if (IsInfo)
```

Comment:
why those `if`s are useful? Normally we use them before calling this method to avoid allocating a string.

Reply:
Are there and strongly typed; simple bool check. Saves on dropping though into all the logging abstractions and interfaces to decide not to log if the caller wasn't guarded